packages/strapi-admin/controllers/Auth.js

```diff
@@ -335,12 +330,7 @@ module.exports = {
       // Send an email to the admin.
       await strapi.plugins['email'].services.email.send({
         to: admin.email,
-        from:
-          settings.from.email || settings.from.name
-            ? `${settings.from.name} <${settings.from.email}>`
-            : undefined,
-        replyTo: settings.response_email,
-        subject: settings.object,
+        subject: 'Reset password',
```

Comment:
I don't understand where all the other params went ? :D

Reply:
They went in the cloud :p

I deleted the hard coded  `from` address to reverse the code introduced by this PR : https://github.com/strapi/strapi/pull/5584